examples/routing_service/shmem_udp_gateway/README.md

```diff
@@ -0,0 +1,172 @@
+# SHMEM <--> UDP Gateway
+
+This repository contains the necessary files to run the RTI Routing Service
+example from the "How does Connext help you meet the constraints of the
+hospital's IT department?" blogpost. Using RTI DDS Ping as a pub/sub
+application and Routing Service you will be able to create a gateway between
+a Shared Memory domain and a UDP domain that uses only 3 ports. There is also
+a second configuration option using RTI Cloud Discovery Service to use Unicast.
+
+## Environment variables
+
+For your convenience, there are 2 scripts to set up environment variables:
+
+- Linux: *variables.sh*
+- Windows: *variables.bat*
+
+These are the variables they contain, which you should modify according to your
+system:
+
+-   **NDDSHOME**: path to the installation of RTI Connext Professional.
```

Comment:
Instead of using NDDSHOME in the instructions, ask the user to run the rtisetenv script. See other examples, for ex:

To build this example, first set all the environment variables as follows:

On *Linux* and *macOS* systems run:

```sh
source <install_dir>/recource/scripts/rtisetenv_<target>.<bash|tcsh|zsh>
```

On *Windows* systems run:

```sh
<install_dir>\resource\scripts\rtisetenv_<target>.bat
```

Reply:
@alexcamposruiz good point. I thought about it, but for this example the only thing that's needed is the *rtiroutingservice* binary, there's no compilation necessary so NDDSHOME is enough. But if you still want me to add it, I will. Just let me know